artistools/packets/__init__.py

```diff
@@ -64,6 +28,77 @@
 type_ids = dict((v, k) for k, v in types.items())
 
 
+@at.diskcache(savezipped=True)
```

Comment:
I'm guessing that this function is not so slow that it's faster to read a zipped file from disk every time. I suggest @lru_cache(maxsize=16) instead.